.eslintrc

```diff
@@ -42,6 +40,9 @@
         "@typescript-eslint/restrict-plus-operands": "off",
         "@typescript-eslint/restrict-template-expressions": "off",
         "@typescript-eslint/unbound-method": "off",
+        "@typescript-eslint/no-unused-expressions": "off",
+        "@typescript-eslint/no-empty-object-type": "off",
+        "@typescript-eslint/no-require-imports": "off",
```

Comment:
Do we need to turn off `no-require-imports`?

Reply:
yeah, there are a lot of require usages all over the codebase